### PR TITLE
Add some links

### PR DESCRIPTION
**Pull request by arsandev**
## Summary:
Add some links to readme.

Added:

### MOOC Web:
- Codepolitan
- Sekolah Koding
- Baledemy
- Belajarpython

### Design
- Build With Angga

### Books
- A Brief Introduction to Machine Learning for Engineers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,12 @@ A collection of awesome useful and awesome links, resources and shiny things for
     * [Curso Em Video](https://www.cursoemvideo.com/)
     * [Estudonauta](https://www.estudonauta.com/)
 
+* Indonesia
+    * [Codepolitan](https://www.codepolitan.com/)
+    * [Sekolah Koding](https://sekolahkoding.com/)
+    * [Baledemy](https://baledemy.com/)
+    * [Belajarpython](https://belajarpython.com/)
```
Comment (oussamabonnor1) [inline README.md:71]: Could you please add some description of these links ? Thank you!

File: README.md [hunk 2]
```diff
@@ -223,9 +235,9 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
     * [Ali Hamdi](https://www.youtube.com/channel/UChiajEj7cSVoY0k2FDlM2zA) Web developement & web design tutorials
     * [Hassouna academy](https://www.youtube.com/user/HassounaAcademy/playlists): Everything related to computer and its sciences, web design and software
     * [Programming with Mosh](https://www.youtube.com/user/programmingwithmosh) programming tutorials
-    
-    
-    
+
```
Comment (oussamabonnor1) [inline README.md:238]: Why the unnecessary line jumps ? 😄

File: README.md (hunk 1 above)
Comment (arsandev) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (arsandev) [reply README.md:238]: Seems like i forgot to tidy up the rows XD. fixed

**Review by oussamabonnor1 — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**